sdk/java-sdk-protobuf-testkit/src/main/java/kalix/javasdk/testkit/KalixProxyContainer.java

```diff
@@ -35,8 +35,8 @@ public class KalixProxyContainer extends GenericContainer<KalixProxyContainer> {
   /** Default user function port (8080). */
   public static final int DEFAULT_USER_FUNCTION_PORT = 8080;
 
-  /** Default local port where the Google Pub/Sub emulator is available (8085). */
-  public static final int DEFAULT_GOOGLE_PUBSUB_PORT = 8085;
+  /** Default local port to use for eventing. */
+  public static final int DEFAULT_EVENTING_PORT = 8999;
```

Comment:
I think defaulting to the internal grpc backend is the right thing to do.. but this change breaks backwards compatibility. Users currently relying on the defaults to use pub/sub externally will now need to either pass config or change the tests to use the new API. Thoughts?